### ISSUE TITLE
Should we show users the progress of their application?

### ISSUE THREAD

**Issue by EUzkuraityte** [alphagov/verify-local-patterns#296]
@sanjaypoyzer @petegale 

I feel like the application is quite lenghty and having gone through it myself I felt a little lost not knowing how far in it I was, how many more steps I needed to complete etc. should we consider user testing this and / or introducing any of these:

1 - progress bar
2 - back button
3 - Previous answers + 'change' option as illustrated below

![screen shot 2017-04-07 at 14 00 19](https://cloud.githubusercontent.com/assets/20285115/24803584/98f033f0-1ba3-11e7-8b34-a47b147f9da0.png)

**Comment by suryanaga** [comment]
@EUzkuraityte I haven't seen anything in user research so far that suggests there's a user need for this.

We've got lots of discussion and research around progress indicators on the hackpad - https://designpatterns.hackpad.com/Progress-indicators-3AOrLoia9Us

I'm adding the question tag to this so we can look out for it in research.

**Comment by joanneSouthern** [comment]
a progress bar or step 1 of X would be good, so you could see how far through you were in the process

**Comment by petegale** [comment]
@joanneSouthern is this something that came up in your research? There are some issues with implementing this, especially when the number of steps varies significantly between different users. If it is a problem we observed, I'd be keen to know where it occurred, as it may well signify a different problem, or different solution.

This is our current guidance on this: https://www.gov.uk/service-manual/design/progress-indicators

**Comment by joanneSouthern** [comment]
yes it came out in research, people just wanting to know how far through they were

**Comment by ingasiler** [comment]
![image](https://cloud.githubusercontent.com/assets/19390851/25745717/32e18fc2-3198-11e7-9b49-ac0822ae6f6e.png)

**Comment by petegale** [comment]
Round 3 showed no evidence of this need, so now closing

**Comment by EUzkuraityte** [comment]
I'm reopening this issue, because:
1. people are complaining that the application is long winded, therefore, we meet the criteria for needing a progress indicator
![screen shot 2017-06-19 at 11 12 31](https://user-images.githubusercontent.com/20285115/27280648-953d54c4-54e0-11e7-8ecf-8bf76486388c.png)
2. People were observed asking for a 'back' button
![screen shot 2017-06-19 at 11 01 09](https://user-images.githubusercontent.com/20285115/27280692-c57e9850-54e0-11e7-888f-608a20528786.png)
I'd suggest adopting GOV.UK version for this
3. we've seen people asking 'When is this [the application] going to end?' - we either need to add a progress indicator to meet this need to know or add a better explanation on the journey upfront (see: #602) or do both

**Comment by lizziebruce** [comment]
Hi, let's talk about this with @sanjaypoyzer and @petegale. 
From a content POV the wording wouldn't be difficult but the content design but would have some issues:
**1. Progress bar**
This would disappear on the Verify hub/IDP part of the journey. Application step 1 of 10, Application step 2 of 10 could be helpful, with obviously no step labels in the Hub. 
But as the others mention above, if different users have different numbers of steps, which they may with PP depending on if free permit and certainly would with CT, it's not ideal. And the confusion of steps disappearing when they hit the hub may outweigh any benefits experienced in the app. The journey is already fairly inconsistent with 3 different sites, having steps that disappear midway through may not add much benefit.
**2. Back button**
It's useful to have a button if the browser Back button breaks the journey or if users aren't noticing it. But if the browser back button works as an in-app one would is there a need to duplicate? 
How many users have asked for an in-app back button - sounds like just one? 
**3. Previous answers**
Putting this on every page may make the application more long-winded, as more items/clutter on page.
The summary page acts as an overview of their previous answers and would be a good place to include links back to earlier pages in the app if they need to make changes.

**Comment by suryanaga** [comment]
Absolutely agree with Lizzie that the fact that we wouldn't be able to make this visible all the way through could make this really tricky and work very well. As far as I understand, the only place people get tired of the process is during the IDP journey and then whizz through the rest of the process after that.

That being said, it seems that some users do have a need to know how far through they are. This doesn't neccesarily mean a progress indicator is the best or only way to meet this need though.

I think it's worth trying something similar to what we have on the concessionary travel start page -- A numbered list outlining the user journey. I'll leave it to @lizziebruce to put something together that we can test.

**Comment by EUzkuraityte** [comment]
moving this to 'ready to do'. See previous comment re: amending the start page to include a numbered summary list of journey  steps

**Comment by lizziebruce** [comment]
@sanjaypoyzer @EUzkuraityte hey both, the alternative prove identity page drafted in #588 could help with this. It flags the steps left from that point. I took out numbers as Irina pointed out that for CT this would be a screen with a list of numbered steps immediately after a screen with a list of numbered steps. But we could do different versions for PP and CT?

**Comment by EUzkuraityte** [comment]
@sanjaypoyzer @lizziebruce yup, the new version should def help.

I'm keen to keep the start page as short as possible, so maybe we could amend the 

> What you'll need
> 
> ability to prove you live in Argleton city centre
> car registration number
> to answer security questions or upload documents to prove who you are

section  instead. And yes, we can definitely have different versions for PP & CT

**Comment by lizziebruce** [comment]
Content fix (done):
6 or 12 months
![screen shot 2017-06-27 at 17 45 07](https://user-images.githubusercontent.com/27814324/27599726-99c5988a-5b61-11e7-9c85-f96426634473.png)

12 months only
![screen shot 2017-06-27 at 17 40 15](https://user-images.githubusercontent.com/27814324/27599742-a05073dc-5b61-11e7-9079-f8f0d4144a36.png)
